fsd_utils/authentication/decorators.py

```diff
@@ -21,6 +23,16 @@ def _failed_redirect():
     return abort(redirect(authenticator_host + signout_route))
 
 
+def _failed_roles_redirect(roles_required: List[str]):
+    authenticator_host = current_app.config[config_var_auth_host]
+
+    return abort(
+        redirect(
+            authenticator_host + roles_error_route + "|".join(roles_required)
```

Comment:
why are we adding the "|" to the redirect path?

Reply:
This is just a separator between the roles_required list values. The roles_error_route ends "..?roles_required=" so this appends "LEAD_ASSESSOR|ASSESSOR" or whatever the roles are that are required.

Reply:
ahh ok awesome 🙌🏾